front-end/src/effects/images.ts

```diff
@@ -17,8 +7,9 @@ const images = {
  * @returns The URL of the image, or `undefined` if the name is invalid.
  */
 export function getImageURL(name: string): string | undefined {
```

Comment:
This is a tricky one. `getImageUrl` is called with a string extracted from a path string. So each call would need a cast from string to `keyof typeof images` since there is no chance that the type-checker finds out if the substring is valid key. So I would keep it the way it is and let the function do the cast.

The other way would be to define an image explicitly instead of extracting it from the path. This would be the cleanest option but requires the most changes.

I'm fine with both options.

Reply:
I see! I guess leave as you proposed.